### PR TITLE
Added a landing page

### PR DESCRIPTION
**Pull request by Aritra777**
# Description

I have added a landing page which is mobile friendly and made with HTML, CSS, Javascript.

Fixes:  #544 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have made this from my own
- [ ] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
This looks wierd.
image

It has to be responsive for all screen sizes. (navbar included). There is a typo in Navbar. It must be "Explore" instead of "Explor". Also, add this project in style_links.json

**Comment by VidhiBhatt01** [comment]
https://deploy-preview-568--cool-front-end-templates.netlify.app/landing%20page/code/

**Comment by Aritra777** [comment]
I have made the changes. Please check it out.
Screenshot 2022-04-04 222928

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Fix the typo too as well as update style_links.json

File: Landing_Page/index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          <div class="logo">
+            <img src="https://cdn.dribbble.com/users/72800/screenshots/3264249/collabtools_logo.png" alt="logo" class="logoImg" />
+            <span class="name">Collab.</span>
+          </div>
+          <ul class="navLinks">
+            <li class="nav-link-items">Features</li>
+            <li class="nav-link-items">Pricing</li>
+            <li class="nav-link-items">Tours</li>
+            <li class="nav-link-items" id="dropDown">
+              Explor
```
Comment (VidhiBhatt01) [inline Landing_Page/index.html:30]: Still not fixed

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]

**Comment by Aritra777** [comment]
@VidhiBhatt01 I have fixed that.

File: Landing_Page/index.html (hunk 1 above)
Comment (VidhiBhatt01) [reply Landing_Page/index.html:30]: "Explore" instead of "Explor"

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]

**Comment by Aritra777** [comment]
@VidhiBhatt01 I have changed the word.

**Review by VidhiBhatt01 — APPROVED** [review]

**Pull request merged**